### PR TITLE
Move resorting of activities to sorted page

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
- sorted page and resorted page share the same mustache template
- page objects are using inheritance to remove duplication

### CODE REVIEW

File: app/locales/en.json [hunk 1]
```diff
@@ -132,11 +132,13 @@
     "activities-heading": "Your activities",
     "activities-text-1": "Get more details about each of the activities - tips, advice and links to useful websites.",
     "activities-text-2": "You can get back to your list from the 'Choose your activities' To-do. Once you've marked that as 'done', there will be a link in your journal that you can use instead.",
-    "activities-text-3": "Re-sort your activities",
+    "resortButton": "Re-sort your activities",
```
Comment (jessstubenbord) [inline app/locales/en.json]: sloppy

**Comment by jessstubenbord** [comment]
Categories remain collapsed on re-sort :sun_with_face:

Back button in the details section takes you back to the sorted list, even if you are coming from the re-sort page :full_moon_with_face:

**Comment by nathancashmoredwp** [comment]
I still like the idea of having a separate controller / page for the re-sort ... as otherwise we are having to manage a lot of if else clauses on the page.

**Comment by lukasz-kaniowski** [comment]
We are having a lot of if/else mostly because we are duplicating the same block 5 times on the page. I can get rid of the if/else code around resort-button. 

In my mind having 2 mustache templates that are 95% identical doesn't sit right. 
The conceptual model here also changed. Before we had dedicated page for resorting and that how it was perceived. Now resorting is perceived as activity on sorted page.

**Comment by lukasz-kaniowski** [comment]
This now has both of the bugs fixed.
I've removed on if/else from mustache template as well. Ready for review.

**Comment by jessstubenbord** [comment]
:last_quarter_moon_with_face: beautiful :first_quarter_moon_with_face:

**Pull request merged**